data/core/docview.lua

```diff
@@ -455,9 +455,14 @@ function DocView:draw_line_text(line, x, y)
   return self:get_line_height()
 end
 
-function DocView:draw_caret(x, y)
-    local lh = self:get_line_height()
+function DocView:draw_caret(x, y, line, col)
+  local lh = self:get_line_height()
+  if self.doc.overwrite then
+    local w = self:get_font():get_width(self.doc.get_char(line, col))
```

Comment:
```suggestion
    local w = self:get_font():get_width(self.doc:get_char(line, col))
```
Small mistake of function call instead of method call.